src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -109,6 +109,32 @@ export const authenticatorsSection = ({
     </aside>`;
 };
 
+export const tempKeysSection = ({
+  authenticators: authenticators_,
+}: {
+  authenticators: Authenticator[];
+}): TemplateResult => {
+  const authenticators = dedupLabels(authenticators_);
+
+  return html` <aside class="l-stack c-card c-card--narrow">
+    <div class="t-title t-title--complications">
+      <h2 class="t-title">Temporary Keys</h2>
+    </div>
+
+    <p style="max-width: 30rem;" class="t-paragraph t-lead">
```

Comment:
why using a `style` and not a class? is that common in the code base?

Reply:
This is analogous to the `authenticatorsSection` above (see line 79).

Also, I'm definitely not the guy to do CSS. 🙃

Reply:
Ok